explainer.md

```diff
@@ -1,22 +1,26 @@
-This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions.
+This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions. This then allows an operating system's file manager or other native UX to select a PWA to handle the file.
 
-This has many use cases. An image editor might be able to display and edit a number of image formats. A movie player might understand a video format. A data visualizer might generate graphs from .CSV files. A Logo environment might interactively step through Logo programs.
+This has many use cases. For example:
+* An image editor could display and edit a number of image formats.
+* A movie player could play a video format.
+* A Javascript environment could interactively step through Javascript programs.
+* A data visualizer could generate graphs from .CSV files.
 
-Some more use cases and motivations are listed in the [Ballista explainer](https://github.com/chromium/ballista/blob/master/docs/explainer.md).
-
-There has historically been no standards-track API for MIME type handling. For some years, Chrome packaged apps have been able to register one or more file handlers using a [Chrome-specific API](https://developer.chrome.com/apps/manifest/file_handlers) where each handler may handle specific MIME types and/or file extensions. This provides some insight into the demand for such functionality. As of August 2018, 619 file handlers handle MIME types, while 509 file handlers handle file extensions. Some of the packages apps have more than one file handler. Overall, 580 packaged apps handle MIME types and 337 packaged apps handle file extensions. This usage, and the use cases above, demonstrate the value of being able to associate web applications with MIME types and/or file extensions.
+There has historically been no standards-track API for MIME type handling. For some years, [Chrome packaged apps](https://developer.chrome.com/docs/extensions/apps/) have been able to register one or more file handlers using a [Chrome-specific API](https://developer.chrome.com/apps/manifest/file_handlers) where each handler may handle specific MIME types and/or file extensions. As of August 2018, 619 file handlers handled MIME types, while 509 file handlers handled file extensions. Some packaged apps had more than one file handler. Overall, 580 packaged apps handled MIME types and 337 packaged apps handled file extensions. This usage, and the use cases above, demonstrates the value of being able to associate web applications with MIME types and/or file extensions.
 
 ### Example
 
-The following web application declares in its manifest that it can handle CSV and SVG files.
+The following web application declares in its manifest that it can handle CSV and SVG files, as well as a hypothetical GRAF file format that may be custom-designed for this application.
```

Comment:
"Hypothetical" allows defining exactly what the format is / how it's used.  Here you could say:

`as well as a hypothetical file format that this application uses called GRAF.`